llpc/context/llpcCompiler.cpp

```diff
@@ -29,14 +29,26 @@
  ***********************************************************************************************************************
  */
 #include "llpcCompiler.h"
+#include "../../lgc/imported/chip/gfx9/gfx9_plus_merged_offset.h"
+#include "../../lgc/imported/chip/gfx9/gfx9_plus_merged_registers.h"
```

Comment:
This looks wrong... @trenouf can probably comment.

Reply:
Yes I agree.  The problem is that those files are not accessible in to llpcCompiler.cpp, but while the linking code still lives in that file, we need access to it to read the metadata.  I should be removed when we move the linking to to lgc.

Reply:
I moved the code that needs these header files to llpcElfWriter.cpp, so we no longer include these in this file.  However, that file has the same issue.  It needs access to these header that are not naturally available.

Reply:
Linking and prolog/epilog generation all needs to be in LGC, not the front-end. I am hoping to look at that next.